ozaria/site/components/teacher-dashboard/BaseTeacherDashboard/TopBanner.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    v-if="showBanner"
+    id="top-banner"
+  >
+    <mixed-color-label
```

Comment:
can we track how many times this was clicked? 

Also, could we hide it after its been clicked?  We could even use localStorage if it's easier. If it's too complicated, we can ignore hiding for now.